lib/decode_jwt.go

```diff
@@ -0,0 +1,65 @@
+package lib
```

Comment:
**Question:** Should we use a more descriptive name?

**Suggestion:** `package lib` -> `package jwt` or `package decode` or `package decode-jwt`

Reply:
renamed